spec/models/shared_ownership_housing_costs_form_spec.rb

```diff
@@ -0,0 +1,120 @@
+require "rails_helper"
+
```

Comment:
Is it worth including some extra tests that the session_data mortgage/rent is actually updated with the correct values?

e.g.  expect(session_contents["shared_ownership_mortgage"]).to eq 500
        expect(session_contents["mortgage_frequency"]).to eq "monthly"

Reply:
added test that covers all of the new fields stored in the session